### ISSUE TITLE
Full e-mail address for invitations is not allowed

### ISSUE THREAD

**Issue by icoloma** [keybase/keybase-issues#439]
I was copying the full e-mail address `John Doe <jdoe@acme.com>` to my invitation and it was not accepted. Shortening `jdoe@acme.com` worked correctly, but this is an easy thing where the program could recover automatically discarding any unused parts.

**Comment by hiergiltdiestfu** [comment]
Nah, this would lead to typos going undetected. I.e. John signs up, accidentially inputs jd<oe@acme.com and the UI silently cuts it down to oe@acme.com. There are quite some issues on this tracker about problems caused by misspelled email addresses. Also, there are different approaches to this "expanded" email address. Exchange does it like John Doe [a@b.c]. 

Just insist on a clean and pure email.

**Comment by icoloma** [comment]
Actually, it's more the other way around: when people use the full e-mail address they are copying and pasting from their contacts or from an exchanged e-mail. Rejecting the data will force the user to manually modify this value, introducing a new chance for error. 

About the Exchange / Outlook format, that's not the standard. You can choose to support both formats, neither, or just the standard. As I said, this is not a killer feature, so it can just be dropped. I only mentioned it as a heads-up.
